.snippets/code/tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet/tests/integration_tests.rs

```diff
@@ -0,0 +1,157 @@
+use custom_pallet::{self, *};
```

Comment:
We should have a license header in all files too, which we should decide on with @albertov19.

Reply:
Sorry, I am not sure what do you mean by "their umbrella"? I found [this information](https://github.com/polkadot-developers/polkadot-docs?tab=readme-ov-file#code-license) in the repository readme which suggests that the license should be `MIT`. Is this correct? @albertov19